src/QirRuntime/samples/StandaloneInputReference/qir-standalone-input-reference.ll

```diff
@@ -0,0 +1,59 @@
+
+%Result = type opaque
```

Comment:
Is it a manually authored LL file? Why does it need to be checked in?

Reply:
Removed the auto-generated file.